packages/unified-selection/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@itwin/unified-selection",
+  "version": "0.0.0",
+  "description": "Package for managing unified selection",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/presentation.git",
+    "directory": "packages/unified-selection"
+  },
+  "keywords": [
+    "Bentley",
+    "EC",
+    "Unified selection",
+    "iTwin.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/unified-selection.js",
+  "module": "lib/esm/unified-selection.js",
+  "types": "lib/cjs/unified-selection.d.ts",
+  "scripts": {
+    "build": "npm run -s build:cjs && npm run -s build:esm",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:esm": "tsc -p tsconfig.esm.json",
+    "build:watch": "npm run -s build:cjs -- -w",
+    "clean": "rimraf lib build",
+    "cover": "",
+    "lint": "eslint ./src/**/*.ts",
+    "test": "mocha --config ./.mocharc.json \"./lib/cjs/test/**/*.test.js\"",
+    "test:watch": "npm -s test -- --reporter min --watch-extensions ts --watch",
+    "docs": "npm run -s docs:extract && npm run -s docs:reference && npm run -s docs:changelog",
+    "docs:changelog": "cpx ./CHANGELOG.md ./build/docs/reference/unified-selection",
+    "docs:reference": "betools docs --includes=./build/docs/extract --json=./build/docs/reference/unified-selection/file.json --tsIndexFile=unified-selection.ts --onlyJson",
+    "docs:extract": "betools extract --fileExt=ts --extractFrom=./src/test --recursive --out=./build/docs/extract",
```

Comment:
I think we should remove docs generation until we have something to generate docs for. 😆